py/setup.py

```diff
@@ -0,0 +1,32 @@
+import os
+import setuptools
+import shutil
+
+# Copies the schema in the module so that setuptools is able to
+# find the file and add it to the package.
+if os.path.isfile("../relay/event.schema.json"):
+    shutil.copyfile(
+        "../relay/event.schema.json",
+        "./sentry_data_schemas/event.schema.json"
+    )
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="sentry-data-schemas",
+    version="0.0.1",
```

Comment:
Shall we be using this version? How do we even version this?

Reply:
This is a question that still requires a follow up. Me and Markus did not yet define the details of the release process for the schema. Likely we should generate a version either at every commit (to the schema, which is not supposed to change that often and in a backward compatible way) or something similar. Anyway more often than "monthly" so that the consumers of the schema (as of now snuba and probably the data pipelines) can get up to date quickly.
Do you see any issue with that ?

In practice we are not even going to publish this to pypi anytime soon. I heard from Armin they are already building the automation needed to publish packages to pypi for other reasons, so waiting for that. Which means this version number will not be very relevant up until then and we will depend (from snuba) on the latest commit for the time being.

Reply:
> Likely we should generate a version either at every commit (to the schema, which is not supposed to change that often and in a backward compatible way) or something similar. Anyway more often than "monthly" so that the consumers of the schema (as of now snuba and probably the data pipelines) can get up to date quickly.
Do you see any issue with that ?

Sounds good to me and also makes me think that we may adopt the CalVer schema that `pytz` uses: https://calver.org/#pytz

Reply:
calver is fine but we would need to adopt a much faster release schedule than onprem and dependencies, since a deploy of this package to sentry.io is always blocked on a release, much like the Relay Python package.

Or we do calver but continue to depend on git hashes internally, but who's the consumer of the calver releases then?

Reply:
We can do whatever we want with CalVer. We can probably do `YY.MM.MICRO` where micro gets incremented until the month flips over. Or we can do what [PYTZ does](https://calver.org/#pytz): `YYYY.MICRO` and keep incrementing the micro version until the year turns.

Reply:
yeah that sounds good

Reply:
regarding that, while we are not publishing this to pypi right away. Do you think of any argument (when we get to publish) against publishing new releases potentially quite often? I would like we generate a version at every commit. I doubt we will ever have frequent commits on this schema (at least I would hope). Though it may happen at times.

Reply:
Strong 👍🏻 for a new package for every new commit as long as we don't publish them to PyPI at this frequency (mostly a courtesy to them, doesn't bother me otherwise 😀 )